ql/src/java/org/apache/hadoop/hive/ql/lockmgr/HiveTxnManager.java

```diff
@@ -60,8 +60,8 @@
   * @return The new transaction id
   * @throws LockException if a transaction is already open.
   */
-  long openTxn(Context ctx, String user, TxnType txnType) throws LockException;
 
+ long openTxn(Context ctx, String user, TxnType txnType, String dbUnderReplication) throws LockException;
```

Comment:
I don't feel the argument `dbUnderReplication` strongly connected to the `openTxn` command - I wonder if it would make sense to move it inside the `Context` object - which could enable to not change the signature; and may also come handy later on

Reply:
Moved to context object.